addon/components/sortable-objects.js

```diff
@@ -35,7 +35,8 @@ export default Ember.Component.extend( {
 
   drop: function(event) {
     if (this.get('enableSort')) {
-      this.sendAction('sortEndAction', event);
+      var obj = this.get('sortableObjectList');
+      this.sendAction('sortEndAction', obj, event);
```

Comment:
@cacqw7 in order to maintain backward compatibility we will need to send obj last.  Also I would appreciate extending one of the tests to ensure this doesn't break in the future.  Though this will be temporary as there will be a change coming in the future that will ensure DDAU and not modify the sortableObjectList directly.